pkg/sdkclient/serverinfo/serverinfo.go

```diff
@@ -31,6 +31,18 @@ import (
 	"github.com/numaproj/numaflow"
 )
 
+// MapModeMetadata field is used to indicate which map mode is enabled
+// If none is set, we consider unary map as default
+const MapModeMetadata = "MAP_MODE"
```

Comment:
Get all the `metadata` keys together in this file? e.g. `MULTIPROC`.

Reply:
Makes sense!